scanner/scanner_test.go

```diff
@@ -248,18 +248,22 @@ func toks(t ...string) []stringTok {
 
 func TestScanner_Scan_scanInString(t *testing.T) {
 	type testCase struct {
-		lit  string
-		toks []stringTok
+		lit      string
```

Comment:
I edited this file pretty heavily to add a bunch of tests related to escaping. I think you can remove all the changes in this file.